src/main/java/de/unistuttgart/chickenshockbackend/data/QuestionDTO.java

```diff
@@ -17,13 +21,21 @@
 @NoArgsConstructor
 @AllArgsConstructor
 @FieldDefaults(level = AccessLevel.PRIVATE)
+@Validated
 public class QuestionDTO {
 
     @Nullable
     UUID id;
 
+    @NotNull(message = "question text cannot be null")
+    @NotBlank(message = "question text cannot be blank")
     String text;
+
+    @NotNull(message = "right answer text cannot be null")
+    @NotBlank(message = "right answer text cannot be blank")
     String rightAnswer;
+
+    @ElementCollection
```

Comment:
WTF? Why on a DTO?
That's a JPA annotation…

Reply:
that was not from me